main.cpp

```diff
@@ -173,143 +173,143 @@ struct CarWash //2)
 
 
 /*
-1)
+1)Audio Interface
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)speaker output
+    2)mic input
+    3)line input
+    4)gain
+    5)phantom power
 3 things it can do:
-    1)
-    2)
-    3)
+    1)amplify sound
+    2)convert analog to digital signal
+    3)invert phase
  */
 
 /*
-2)
+2)Tree
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)leaves
+    2)seeds
+    3)water
+    4)flowers
+    5)energy
 3 things it can do:
-    1)
-    2)
-    3)
+    1)photosyntesis
+    2)releases oxygen
+    3)produces fruits
  */
 
 /*
-3)
+3)Dog
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)legs
+    2)fur
+    3)eyes
+    4)nose
+    5)tail
 3 things it can do:
-    1)
-    2)
-    3)
+    1)bark
+    2)eat
+    3)jump
  */
 
 /*
-4)
+4)Monitor
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)woofer
+    2)tweeter
+    3)rca input
+    4)line input
+    5)aux input
 3 things it can do:
-    1)
-    2)
-    3)
+    1)Receives signal
+    2)Play sound
+    3)Change volume
  */
 
 /*
-5)
+5)paper tray
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)plastic
+    2)rigid body
+    3)edge guides
+    4)sheet feeder
+    5)input tray shield
 3 things it can do:
-    1)
-    2)
-    3)
+    1)hold paper
+    2)opens
+    3)counts amount of paper
  */
 
 /*
-6)
+6)display
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)pixels
+    2)plasma
+    3)sensitivity
+    4)touch integrated
+    5)power circuit
 3 things it can do:
-    1)
-    2)
-    3)
+    1)show menu
+    2)show error
+    3)change option
  */
 
 /*
-7)
+7)print heads
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)print heads latch
+    2)chip
+    3)ink chamber
+    4)filter
+    5)screws
 3 things it can do:
-    1)
-    2)
-    3)
+    1)receives instructions
+    2)prints
+    3)calibrates ink noozles
  */
 
 /*
-8)
+8)ink tank
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)container
+    2)ink color
+    3)hemetic
+    4)ink level
+    5)tank cap
 3 things it can do:
-    1)
-    2)
-    3)
+    1)checks quality of ink
+    2)counts amount of ink
+    3)warns if ink is low
  */
 
 /*
-9)
+9)scanner
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)document table
+    2)document cover
+    3)carriage
+    4)lamp
+    5)lid
 3 things it can do:
-    1)
-    2)
-    3)
+    1)scan document
+    2)makes copies
+    3)re-size documents
  */
 
 /*
-10)
+10)Printer
```

Comment:
I think you mean `Office Copier`.  most printers can't scan, unless they are a scanner-printer combo device.